src/en/community/community.json

```diff
@@ -1,5 +1,41 @@
 {
   "tech_talks": [
+    {
+      "date": "2021-03-10",
+      "title": "What's New in Pacific Ceph Release",
+      "presenter": "Sage Weil",
+      "url": "https://www.youtube.com/watch?v=PVtn53MbxTc"
+    },
```

Comment:
should this go at the end so that it's is in reverse chronological order?

Reply:
There's a line in index.html that looks like this:

{% set eventLocaleTag = locale + '-event' %} {% set eventsFuture = collections[eventLocaleTag] | getItemsInFuture | reverse | getItems(4) %}

This line automatically sorts the json objects in community.json into reverse chronological order when the website is rendered.

However (and I don't mean this in a petulant way), if you'd prefer the .json file itself to have these objects sorted by human hands into chronological order, that is a task well within my vim competencies.

Let me know if you want me to hand-sort the json file.

Reply:
@zdover23 If you want to hand-sort the json file that's fine. It will definitely be easier to edit and maintain if it reads the same way as it renders in the site. I'll leave the decision to you as it's not essential for rendering due to the `reverse` filter.

Reply:
I've sorted them.

Reply:
:+1: